package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/vision",
   "description": "Google Cloud Vision API client for Node.js",
-  "version": "0.20.0",
+  "version": "0.21.0",
```

Comment:
There should be a corresponding change in the `package-lock.json` as well.

I typically use npm to do the version bump for me, and it takes care of updating both.

```
$ npm version --no-git-tag-version {minor|major|patch|specific version}
```

We use the no-git-tag as per the release process at go/npm-at-google wherein the release is published when the draft release notes are published.

Reply:
Cool, shouldn't be a problem to do that. Moved this over to https://github.com/GoogleCloudPlatform/releasetool/pull/6#issuecomment-401478834